netlogo-gui/src/main/app/codetab/CodeTab.scala

```diff
@@ -50,7 +54,20 @@ class CodeTab(val workspace: AbstractWorkspace) extends JPanel
   override def zoomTarget = text
 
   val errorLabel = new EditorAreaErrorLabel(text)
-  val toolBar = getToolBar
+  val toolBar = new ToolBar {
```

Comment:
Here are the gains:

1) Removes clutter from initializer
2) Overriding a `val`s is tricky, especially if the `val` is acted upon in the initializer.
3) Overriding `val` leads both the subclass and the superclass to run their definitions, resulting in extra computation and an object which is never used.